codalab/lib/bundle_cli.py

```diff
@@ -1326,7 +1326,7 @@ def display_interpreted(self, client, worksheet_info, interpreted):
                         try:
                             self.print_target_info(client, data, decorate=True, maxlines=maxlines)
                         except UsageError, e:
-                            print 'ERROR:', e
+                            print 'MISSING'
```

Comment:
This is error-prone, because a UsageError doesn't necessarily mean 'MISSING' in all cases (even though currently it might be the case).  I think the better way to do all this is to add an additional argument to print_target_info called 'fail_if_not_exist', which is only set to true in do_cat_command.